### DISCUSSION TITLE
윈브 dpm이 더 낮아졌습니다.

### DISCUSSION THREAD

**Discussion by mechmango** [oleneyl/maplestory_dpm_calc#651] [Q&A] [answered]
천노 오류와 게일 스택화 및 크리인,소울컨트랙트 최적화를 진행한걸로 아는데 천노오류를 수정하지 않은걸 기준으로 계산해도 이전보다 딜량이 더 감소했습니다.  제가 가지고있던 버전이 너무 오래된걸까요? 
개인적인 생각으로는 크리인 쿨이 30초이상 남은경우 게일 스택을 쌓는다고 했는데 이 과정에서 2스택이 쌓여도 크리인 쿨이 남는 경우가 생겨서 그런것 같습니다.

**Comment by icepeng** [comment, marked as answer]
dpm표에 게시된 자료는 쿨감 4초 기준입니다. --cdr 4를 주고 돌려보시면 이전보다 약간 상승한 것을 확인할 수 있습니다.

**Reply by mechmango** [reply]
쿨감 4초기준으로 돌린건데도 감소했던데 제 버전이 좀 예전껀가보네요

**Reply by mechmango** [reply]
제가 설정을 잘못했었네요 정상적으로 증가해서 나오는것같습니다.
